Source/GoogleBiddingAdapterConfiguration.swift

```diff
@@ -10,17 +10,33 @@ import os.log
 /// A list of externally configurable properties pertaining to the partner SDK that can be retrieved and set by publishers.
 @objc public class GoogleBiddingAdapterConfiguration: NSObject {
 
+    /// The version of the partner SDK.
+    @objc public static var partnerSDKVersion: String {
+        let versionNumber = GADMobileAds.sharedInstance().versionNumber
+        return "\(versionNumber.majorVersion).\(versionNumber.minorVersion).\(versionNumber.patchVersion)"
+    }
+
+    /// The version of the adapter.
+    /// It should have either 5 or 6 digits separated by periods, where the first digit is Chartboost Mediation SDK's major version, the last digit is the adapter's build version, and intermediate digits are the partner SDK's version.
+    /// Format: `<Chartboost Mediation major version>.<Partner major version>.<Partner minor version>.<Partner patch version>.<Partner build version>.<Adapter build version>` where `.<Partner build version>` is optional.
+    @objc public static let adapterVersion = "4.11.2.0.0"
+
+    /// The partner's unique identifier.
+    @objc public static let partnerID = "google_googlebidding"
+
+    /// The human-friendly partner name.
+    @objc public static let partnerDisplayName = "Google bidding"
+
     private static let log = OSLog(subsystem: "com.chartboost.mediation.adapter.google_googlebidding", category: "Configuration")
 
     /// Google's identifier for your test device can be found in the console output from their SDK
-    class func setTestDeviceId(_ id: String?) {
-        if let id = id {
-            GADMobileAds.sharedInstance().requestConfiguration.testDeviceIdentifiers = [id]
-        } else {
-            GADMobileAds.sharedInstance().requestConfiguration.testDeviceIdentifiers = []
+    @objc public static var testDeviceIdentifiers: [String]? {
+        get {
+            GADMobileAds.sharedInstance().requestConfiguration.testDeviceIdentifiers
         }
-        if #available(iOS 12.0, *) {
-            os_log(.debug, log: log, "Google Bidding SDK test device ID set to %{public}s", id ?? "nil")
+        set {
+            GADMobileAds.sharedInstance().requestConfiguration.testDeviceIdentifiers = newValue
+            os_log(.debug, log: log, "AdMob SDK test device ID set to %{public}s", newValue ?? "nil")
```

Comment:
It was `String?` but now `[String]?`, so I think the log message
`"AdMob SDK test device ID set to %{public}s"`
should be
`"AdMob SDK test device IDs set to %{public}"`